typescript/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "sourceMap": true,
+    "declaration": true,
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "emitDecoratorMetadata": true,

```

Comment:
I think we generally need ``"resolveJsonModule": true``, we have this use case on various libraries.

Reply:
Hmm yeah, this seems useful. I will add it 👍

Reply:
Thanks!

Reply:
DONE